### ISSUE TITLE
BREAKing out of GWBASIC programs, disk images, USB drive support, CTRL-ALT-DEL, and flashing text in GWBASIC.

### ISSUE THREAD

**Issue by JmE13** [jhhoward/Faux86#2]
I am newly registered at github and did so specifically because of Faux86. (I have found no other way of contacting you except here so if this is not the place, I apologize.) It's quite impressive and exactly what I was looking for; a bare metal PC emulator for my Raspberry Pi Zero W. This year, I plan on embedding the Zero in a keyboard for a nostalgia MS-DOS/GWBASIC machine. 

1) I had some trouble with breaking out of simple GWBASIC programs. I tried CTRL-BREAK, CTRL-C, and BREAK, Nothing seemed to do the trick.
2) Is CTRL-ALT-DEL going to be implemented at some point?
3) Is there some way to make your own IMG files? With AUTOEXEC.BAT and CONFIG.SYS on the A: IMG, it seems like it would be impossible to edit those files. I would like to be able to install full DOS 6.22 on the hard drive image and boot from there.
4) Would implementing a USB floppy drive be possible? That might help.
5) Is USB flash drive support in the future plan?
6) I think the video emulation is missing something as I could not do flashing text under GWBASIC.

Thanks for taking the time to create this emulator. It is very good! I am really enjoying it.

**Comment by Kingeorg** [comment]
Hello.
According to your words above:
1. You made it work (compiled it successfully).
2. You are looking to achieve a bare-metal MS-DOS machine just like I do.
3. I would also like to run MS-DOS from the SD (hard drive).

Can you kindly walk me through as how to compile it? Have you had any success in trying to install DOS 6.22?
I ask you because you seem to have achieved the most success towards realizing a my dream of a Raspberry Pi Zero DOS machine.
Thank you.

**Comment by ArnoldUK** [comment]
Yes it is possoble to create your own bootable .IMG disks. I use [WinImage](https://www.winimage.com/winimage.htm) I've been using Faux86 on a RPi 3 on and off for a few years now but not much has changed in the source by way of updates over the past few years. There are a few issues with compatibility in native DOS.
